services/state/addressbook/node.proto

```diff
@@ -27,81 +25,202 @@ option java_package = "com.hederahashgraph.api.proto.java";
+     * <p>
+     * These endpoints SHALL represent the published endpoints to which clients
+     * may submit transactions.<br/>
+     * These endpoints SHOULD specify address and port.<br/>
+     * These endpoints MAY specify a DNS name.<br/>
+     * These endpoints SHOULD NOT specify both address and DNS name.<br/>
+     * This list SHALL NOT be empty.<br/>
+     * This list SHALL NOT contain more than `8` entries.
+     * <p>
+     * Each network may have additional requirements for these endpoints.
+     * A client MUST check network-specific documentation for those
+     * details.
      */
-    repeated ServiceEndpoint service_endpoint = 5;
+    repeated proto.ServiceEndpoint service_endpoint = 5;
 
     /**
-     * The node's X509 certificate used to sign stream files (e.g., record stream
-     * files). Precisely, this field is the DER encoding of gossip X509 certificate.
+     * A certificate used to sign gossip events.
+     * <p>
+     * This value SHALL be a certificate of a type permitted for gossip
+     * signatures.<br/>
+     * This value SHALL be the DER encoding of the certificate presented.<br/>
+     * This field is REQUIRED and MUST NOT be empty.
      */
     bytes gossip_ca_certificate = 6;
 
     /**
-     * node x509 certificate hash. Hash of the node's TLS certificate. Precisely, this field is a string of
-     * hexadecimal characters which, translated to binary, are the SHA-384 hash of
-     * the UTF-8 NFKD encoding of the node's TLS cert in PEM format. Its value can be
-     * used to verify the node's certificate it presents during TLS negotiations.
+     * A hash of the node gRPC certificate.
+     * <p>
+     * This value MAY be used to verify the certificate presented by the node
+     * during TLS negotiation for gRPC.<br/>
+     * This value MUST be a SHA-384 hash.<br/>
+     * The TLS certificate to be hashed SHALL first be in PEM format and SHALL
+     * be encoded with UTF-8 NFKD encoding to a stream of bytes provided to
+     * the hash algorithm.<br/>
+     * This field is OPTIONAL.
      */
     bytes grpc_certificate_hash = 7;
 
     /**
-     * The consensus weight of this node in the network.
+     * A consensus weight.
+     * <p>
+     * Each node SHALL have a weight in consensus calculations.<br/>
+     * The consensus weight of a node SHALL be calculated based on the amount
+     * of HBAR staked to that node.<br/>
+     * Consensus SHALL be calculated based on agreement of greater than `2/3`
+     * of the total `weight` value of all nodes on the network.
      */
     uint64 weight = 8;
 
-    /**
-     * an enum to indicate the status of the node, not used in phase 2
+    reserved "status";
+    reserved 9;
+    /* [Placed AFTER reserved, and not doc-comment due to PBJ issue #217]
+     * Reserved as `NodeStatus status = 9;`
+     * A node status value.<br/>
+     * This value is not yet used and design is incomplete.
      */
-    //NodeStatus status = 9;
 
 }
 
+/**
```

Comment:
We should not define `NodeStatus` at this point. The design for the next phase is incomplete, and one can easily imagine a solution that does not need a `NodeStatus`.

Reply:
What you are requesting is a non-functional design.  Without *something* we have no pending state and much of the rest of the design is impossible to implement.  That suggests that *this phase* design is incomplete, in which case we need more work and should consider the need for some form of status(or alternative) as part of the current phase.

Reply:
Note, if we have a solid design that will definitely work _as specified_ for phase 2; I have no issue to remove this pre-existing definition entirely. If we intend to keep this in the protobuf definitions, however, it needs to be well specified.

Side note. If we have a design that works but is _different_ from specification, then, in my opinion, we need to **publicly document** that design and then we can update specification accordingly.

Reply:
Status field and enum are removed.